src/test/storage/constraints_test.cpp

```diff
@@ -363,41 +321,20 @@ TEST_F(ConstraintsTest, ValidInsertDeleteRace) {
   EXPECT_FALSE(delete_op->execute_failed());
   EXPECT_TRUE(del_transaction_context->commit());
 
-  // add new values
-  auto gt = std::make_shared<GetTable>("new_values");
-  gt->execute();
-  auto insert = std::make_shared<Insert>("table", gt);
-  auto insert_context = TransactionManager::get().new_transaction_context();
-  insert->set_transaction_context(insert_context);
-  insert->execute();
+  auto[insert, insert_context] = _insert_values("table", new_values);
 
   EXPECT_FALSE(insert->execute_failed());
   EXPECT_TRUE(insert_context->commit());
 }
 
 TEST_F(ConstraintsTest, InsertInsertRace) {
-  auto& manager = StorageManager::get();
-  auto table = manager.get_table("table");
   auto new_values = std::make_shared<Table>(column_definitions, TableType::Data, 2, UseMvcc::Yes);
-  manager.add_table("new_values", new_values);
-  new_values->append({5, 0, 1});
-
-  auto gt = std::make_shared<GetTable>("new_values");
-  gt->execute();
-
-  // Both operators want to insert the same value
-  auto insert_1 = std::make_shared<Insert>("table", gt);
-  auto insert_2 = std::make_shared<Insert>("table", gt);
-
-  auto insert_1_context = TransactionManager::get().new_transaction_context();
-  auto insert_2_context = TransactionManager::get().new_transaction_context();
-  insert_1->set_transaction_context(insert_1_context);
-  insert_2->set_transaction_context(insert_2_context);
+  new_values->append({5, 42, 1, 42});
 
   // They both execute successfully since the value was not commited by either of them at the point of execution
-  insert_1->execute();
+  auto[insert_1, insert_1_context] = _insert_values("table", new_values);
```

Comment:
format?

Reply:
Agreed, it's weird. We introduced it in 9d6c32c659deb6e2a7befadf2dd5e7916e296706 because [apparently the linter does not like a space there.](https://hyrise-ci.epic-hpi.de/blue/organizations/jenkins/hyrise%2Fhyrise/detail/PR-1495/2/pipeline/36)